frontend/src/components/CloseSuspendReasonModal.css

```diff
@@ -0,0 +1,30 @@
+.smart-hub--goal-close-suspend-reason .usa-form-group fieldset textarea {
+    resize: none;
+    width: 415px;
+}
+
+.usa-modal__footer .usa-button-group__item .usa-button--unstyled {
+    padding-left: 8px;
+    padding-right: 8px;
+    margin-right: 8px;
+    margin-left: 8px;
+}
```

Comment:
functionally the same as padding-x-1 margin-x-1, can we use the usdws classes instead?

Reply:
Whats being selected here is in the base modal class. We only want this change for the close/suspend modal. 

Let me know if you think there is a better way to do this in CSS.

Reply:
Added close/suspend modal prefix in css.